sirepo/package_data/static/css/sirepo.css

```diff
@@ -897,6 +897,7 @@ li.sr-dropdown-submenu > a {
   overflow-y: auto;
   border: 1px solid #e7e7e7;
   min-height: 80ex;
+  max-height: calc(100vh - 65px);
   border-radius: 4px;
```

Comment:
I change the css a bit to use a calc() and directly on the sr-sidebar class. 
![x](https://github.com/radiasoft/sirepo/assets/6516910/90687f25-e00f-4fdf-ad46-a49b22198d08)